Samples/Android/PlayFabCommerce/Readme.md

```diff
@@ -0,0 +1,34 @@
+PlayFabCommerce sample for Android
+=
+
+This sample uses Android's in-app billing API and PlayFab client API to demonstrate an in-app store that exchanges real money for virtual items in PlayFab Economy.
+
+## Installing the sample
+
+The app can be deployed from Android Studio to a compatible AVD, supporting SDK level 21 or above and using Google Play services (not just Google API). This is because signin with a valid Google account is required. It can also be deployed to any physical Android Device meeting the SDK requirements (5.0/Lollipop).
+
+Due to signing requirements, the deployed app may not successfully launch the real money billing flow, resulting in responseCode 4 for onPurchasesUpdated(). If this occurs, try installing the Play Store version prior to deploying the development version.
+
+The app can also be installed from the Play Store after joining the [PlayFabCommerce test group][1]. After joining (approval required, please contact [ATG Marketplace team][2]), go to the [opt-in link][3] and follow the steps to install the app on your device.
```

Comment:
Can we get a different alias for this?  I would prefer if atgstore wasn't publicly broadcasted.